packages/filesystem/src/node/filesystem-backend-module.ts

```diff
@@ -36,18 +34,13 @@ export function bindFileSystem(bind: interfaces.Bind, props?: {
 }
 
 export function bindFileSystemWatcherServer(bind: interfaces.Bind): void {
-    if (cluster.isMaster) {
```

Comment:
We should preserve `--no-cluster` for single thread debugging.

Reply:
Perhaps we should rename it as `--single-thread`?

Reply:
i would prefer to keep the name to avoid breaking existing launch configuration, i know that it is used by different clients for debugging

Reply:
So you want these sections to be debugged by default in a single threaded way, although production doesn't run like that and you want the name kept referencing clustering although it isn't?

Reply:
I meant if there is `--no-cluster` option then fs and git watching should happen in own processes. If there is then they should not. Something like `process.argv.indexOf('--no-cluster') !== -1` for backward compatibility in launch configurations.